scripts/output/projects/FSEC_dietaryIndicators.R

```diff
@@ -0,0 +1,51 @@
+# |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# --------------------------------------------------------------
+# description: Extract population-level dietary information and consumption data from a MAgPIE run
+# comparison script: FALSE
+# ---------------------------------------------------------------
+
+# Version 1.00 - Michael Crawford
+# 1.00: first working version
+
+library(lucode2)
+library(magpie4)
+
+print("Start FSEC dietary indicators output runscript")
```

Comment:
```suggestion
message("Start FSEC dietary indicators output runscript")
```
I would recommend to use `message` instead of `print` for status messages